src/main.cpp

```diff
@@ -26,10 +26,29 @@ VirtualTimerGroup read_timer;
 CANSignal<uint8_t, 0, 8, CANTemplateConvertFloat(1), CANTemplateConvertFloat(0), false> cur_throttle_signal{};
 
 // TX CAN Message (Might need to edit transmit period (currently 100))
-// CANTXMessage<1> tx_message{can_bus, 0x300, 1, 100, read_timer, cur_throttle_signal};
+CANTXMessage<1> tx_message{can_bus, 0x300, 1, 100, read_timer, cur_throttle_signal};
+
+CANSignal<float, 0, 16, CANTemplateConvertFloat(0.1), CANTemplateConvertFloat(-40), false> motor_temp_signal{};
+CANSignal<float, 16, 16, CANTemplateConvertFloat(0.1), CANTemplateConvertFloat(-40), false> coolant_temp_signal{};
+CANSignal<float, 32, 16, CANTemplateConvertFloat(0.1), CANTemplateConvertFloat(-40), false> ambient_temp_signal{};
+CANRXMessage<3> rx_message{can_bus, 0x420, motor_temp_signal, coolant_temp_signal, ambient_temp_signal};
 
 void ReadAcceleratorPress() {
 	cur_throttle_signal = throttle.ReadAcceleratorPress();
+	Serial.print("cur_throttle_signal: ");
+	Serial.println(cur_throttle_signal);
+	Serial.println("\n");
+};
+
+void printReceiveSignals() {
```

Comment:
consistency with function names (by google c++ standards, should be pascal case)